scripts/setup-mapbox-sdk-walkthrough.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo -e "This script helps you set up credentials needed to use Mapbox SDKs for iOS and Android\n\nWe need two kinds of tokens: one for downloading Mapbox's closed-sourced SDKs, and the other for accessing Mapbox's API from these SDKs"
+
+echo -e "\nStep 1: Go to https://account.mapbox.com/access-tokens/. If you don't have a Mapbox account yet, crease one. Copy the token named \"Default public token.\" This token is used to call Mapbox API to display maps and get directions. Paste the copied token below and hit enter (the token won't be displayed)\n"
+
+read -s -p "Default public token: " PUBLIC_TOKEN
```

Comment:
Can we remove this? We have the MapboxToken manager library for this purpose. Also it looks like you're not doing anything with it here.

Reply:
ah forgot to remove that 😓